bigbone/src/main/kotlin/social/bigbone/api/entity/Preferences.kt

```diff
@@ -0,0 +1,49 @@
+package social.bigbone.api.entity
+
+import com.google.gson.annotations.SerializedName
+import social.bigbone.api.entity.data.Privacy
+
+/**
+ * Represents a Preferences about an account.
+ * @see <a href="https://docs.joinmastodon.org/entities/Preferences/">Mastodon API Preferences</a>
+ */
+data class Preferences(
+    /**
+     * Default visibility for new posts.
+     */
+    @SerializedName("posting:default:visibility")
```

Comment:
When merging latest changes from the main branch back in, please change the `@SerializedName` to `@SerialName` and add `@Serializable` to both `data class Preferences` and `enum class Media`

Reply:
Many thanks for alerting me about

Reply:
I do not think I have to add anything to the Media Enum, right?

Reply:
You may not need to. If you add it to the top level class, Android Studio probably complains something is missing if you’d need to add it.

At the latest, you’ll notice in unit tests if you access anything of type `Media`